docs/src/main/asciidoc/security-openid-connect-client-reference.adoc

```diff
@@ -5,14 +5,17 @@ https://github.com/quarkusio/quarkus/tree/main/docs/src/main/asciidoc
 ////
 = OpenID Connect (OIDC) and OAuth2 Client and Filters Reference Guide
 include::_attributes.adoc[]
+:diataxis-type: reference
 :categories: security
 :topics: security,oidc,client
 :extensions: io.quarkus:quarkus-oidc-client
 
-This reference guide explains how to use:
+This guide covers Quarkus extensions for OpenID Connect/OAuth 2.0 token management, focusing on acquiring, refreshing, and propagating tokens with Keycloak, crucial for Quarkus app authentication and authorization.
```

Comment:
```suggestion
This guide covers Quarkus extensions for OpenID Connect and OAuth 2.0 token management, focusing on acquiring, refreshing, and propagating tokens with Keycloak, crucial for Quarkus app authentication and authorization.
```

OIDC and OAuth2 are not equivalent so `/` does not quite suit IMHO

Reply:
Updated. Thanks for the suggested text.